src/cli/commands/files/cat.js

```diff
@@ -31,13 +31,12 @@ module.exports = Command.extend({
         })
         return
       }
-      ipfs.files.cat(path, (err, res) => {
+      ipfs.files.cat(path, (err, file) => {
         if (err) {
           throw (err)
         }
-        res.on('data', (data) => {
-          data.stream.pipe(process.stdout)
-        })
+        console.log(file)
+        file.pipe(process.stdout)
```

Comment:
This is on the CLI, which is not covered by the interface-ipfs-core tests. If you check the cli-test, you are not checking the output of the CLI.